manifests/check.pp

```diff
@@ -147,17 +147,17 @@
 
   case $::osfamily {
     'windows': {
-      $etc_dir = 'C:/opt/sensu'
-      $conf_dir = "${etc_dir}/conf.d"
-      $user = undef
-      $group = undef
+      $etc_dir   = 'C:/opt/sensu'
+      $conf_dir  = "${etc_dir}/conf.d"
+      $user      = $::sensu::user
```

Comment:
If this is not strictly needed, it would be better in the future to separate out any unneeded changes. This makes the PR easier to follow and test as now this PR will impact Windows. You can leave it here though, as I'll be testing all the OS's anyhow with this big change.